src/main/java/dk/sdu/seb05/semesterprojekt/DomainLayer/DomainController.java

```diff
@@ -25,14 +36,19 @@
         return iProducer.getProgrammes();
     }
 
+    @Override
+    public List<IProgramme> getProgrammes(){
+        return programmes;
+    }
+
     @Override
     public List<IProducer> getProducers(){
-        return dataLayer.getProducers();
+        return producers;
```

Comment:
This change might lead to a problem with that producers array getting desynced from the "real" producers in the DataLayer when things are changed. I'm not just, but it has to be tested.

Reply:
Har ændret det tilbage til `dataLayer.get***();` de steder hvor der blev brugt Lists for at være sikker på der ikke sker desync